rabbitmq/datadog_checks/rabbitmq/openmetrics/metrics.py

```diff
@@ -0,0 +1,438 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+
+_COUNTS = {
+    "erlang_mnesia_committed_transactions": "erlang.mnesia.committed_transactions",
+    "erlang_mnesia_failed_transactions": "erlang.mnesia.failed_transactions",
+    "erlang_mnesia_logged_transactions": "erlang.mnesia.logged_transactions",
+    "erlang_mnesia_restarted_transactions": "erlang.mnesia.restarted_transactions",
+    "erlang_vm_msacc_alloc_seconds_total": "erlang.vm.msacc.alloc_seconds",
```

Comment:
Does this metric get collected? Curious since for counts, we need to remove the `_total`

Reply:
Ahh nvm. It gets removed in the `_RENAME_RABBITMQ_TO_DATADOG` object

Reply:
Yup, it does get collected. There's a function that strips the `_total` from the raw metric names before renaming